src/sandboxes/web3-react-v6-sandbox/README.md

```diff
@@ -0,0 +1,5 @@
+# Phantom Wallet Sandbox
+
+> A CodeSandbox for learning how to interact with Phantom Wallet
+
+[Play with the sandbox in full view](https://36ll5s.csb.app/)
```

Comment:
Let's remove these links for now, I think they point to old ETH sandboxes?

Reply:
Good catch